tensorflow/python/platform/gfile.py

```diff
@@ -382,13 +382,15 @@ def ListDirectory(directory, return_dotfiles=False):  # pylint: disable=invalid-
   return files
 
 
-def Walk(top, topdown=1, onerror=None):
+def Walk(top, topdown=1, onerror=None, followlinks=False):
```

Comment:
This was also referenced by Dan in https://github.com/tensorflow/tensorflow/issues/2140#issuecomment-215888562, so I guess that's another vote :)